code/modules/vehicles/multitile/cm_armored.dm

```diff
@@ -55,7 +55,7 @@ GLOBAL_LIST_INIT(armorvic_dmg_distributions, list(
 		"abstract" = 1.0) //abstract for when you just want to hurt it
 
 	//Decisecond cooldowns for the slots
-	var/list/cooldowns = list(
+	cooldowns = list(
```

Comment:
avoid the conflict with the item level cooldowns

Reply:
Might want to change its name, since this uses a different system and will pollute the other one, no?